index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <a href="#welcome-section">Welcome</a>
+      <a href="#projects">Projects</a>
+      <a id="profile-link" target="_blank" href="https://github.com/austync/Augustine-and-Arnaud-Team">My github</a>
+    </nav>
+    <section id="welcome-section">
+      <h1>Augustine and Arnaud</h1>
+    </section>
+    <section id="projects">
+      <h2 class="project-tile">First project</h2>
+      <a href="https://www.delftstack.com/howto/javascript/javascript-max-of-array/">Link to my portfolio</a>
+    </section>
+
+
+
```

Comment:
too many unused spaces. i think it should they should be removed for cleaner code

Reply:
Example: 
Nice job on your code but I'll recommend removing unnecessary space in your code. By doing so you'll make your code easier to read and debug for you and your collaborators

Reply:
You did a great job on your code but i will recommend you remove the unnecessary spaces in your code. By doing this you will make your code easier to read and you will also make debugging of your code easy.